flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/sinks/OrionSink.java

```diff
@@ -61,18 +61,18 @@
 public abstract class OrionSink extends AbstractSink implements Configurable {
 
     private Logger logger;
-    
```

Comment:
Do not know why this deletions and insertions... maybe something related with environment used by @crbrox , who created the branch.